utils/featureFlagHelpers.ts

```diff
@@ -0,0 +1,7 @@
+import { getFeatureFlags } from '../features';
+
+export const isFeatureFlagActive = (featureName: string) => {
+  const features = getFeatureFlags();
+
+  return features[featureName].isActive;
```

Comment:
In our hook and component we have some more advanced checks, specifically one that checks if the `featureName` exists in `features`. We should probably look to re-use or replicate that here.

I'm wondering if it actually makes sense for this `isFeatureFlagActive` function to live with all the other feature flags stuff?

Reply:
Thanks 🙌🏾 . 
I wasn't sure if it should sit with others, but it's probably clearer to have them all together 👍🏾

Reply:
Done in 8608e2b